package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@mitodl/smoot-design",
+  "version": "0.0.0",
+  "packageManager": "yarn@4.5.1",
+  "scripts": {
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build --docs",
+    "test": "jest",
+    "typecheck": "tsc --noEmit",
+    "build:esm": "tsc",
+    "build:cjs": "tsc --module commonjs --outDir dist/cjs",
+    "build": "rm -rf dist && rm -f .tsbuildinfo && npm run build:esm && npm run build:cjs",
+    "lint-check": "eslint src/ .storybook/",
+    "lint-fix": "yarn lint-check --fix",
+    "fmt-check": "prettier --ignore-path .gitignore --ignore-path .prettierignore --check .",
+    "fmt-fix": "prettier --ignore-path .gitignore --ignore-path .prettierignore --write ."
+  },
+  ".": {
+    "import": "./dist/esm/index.js",
+    "require": "./dist/cjs/index.js",
+    "default": "./dist/esm/index.js"
+  },
```

Comment:
These need to go in "exports". It's helpful (I read [here](https://www.kravchyk.com/typescript-npm-package-json-exports/), and [here](https://stackoverflow.com/a/77566206/1181545)) to include the typings, ie. 
```
  "exports": {
    ".": {
      "import": {
        "types": "./dist/esm/index.d.mts",
        "default": "./dist/esm/index.js"
      },
      "require": {
        "types": "./dist/cjs/index.d.mts",
        "default": "./dist/cjs/index.js"
      },
      "default": {
        "types": "./dist/esm/index.d.mts",
        "default": "./dist/esm/index.js"
      }
    }
  },
```